README.md

```diff
@@ -1,2 +1,3 @@
 # Flower orders in Vue
 Vue simple project to get flower orders
+YES!
```

Comment:
YES

Reply:
That's my contribution to the project. (proud)